daemon/daemon.go

```diff
@@ -938,6 +938,7 @@ func NewDaemonFromDirectory(config *Config, eng *engine.Engine, registryService
 
 	if !config.DisableNetwork {
 		if err := bridge.InitDriver(&config.Bridge); err != nil {
+			logrus.Errorf("Error initializing Bridge: %s", err)
```

Comment:
I think this should be `return nil, fmt.Errorf("Error initializing Bridge: %v", err)` (note the %v).

The handler at the top level can then show where exactly this is coming from.